Tangem/App/Utilities/MarketCapFormatter/CommonMarketCapFormatter.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  CommonMarketCapFormatter.swift
+//  Tangem
+//
+//  Created by skibinalexander on 19.06.2024.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+struct CommonMarketCapFormatter {
```

Comment:
думаю `Common` тут можно опустить, т.к. протокола нет, да и в целом не надо, если появятся какие-то другие форматтеры, они тогда будут называться `...MarketCapFormatter`
```suggestion
struct MarketCapFormatter {
```